<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="welcomeLbl">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="NameOfFirst">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>148</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QSpinBox" name="tweetsCount">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>33</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>124</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Search Term</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>How Many tweets you want</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="GetTweets">
      <property name="minimumSize">
       <size>
        <width>93</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Get Tweets</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="2" colspan="4">
     <widget class="QTextEdit" name="mytextEdit_8">
      <property name="minimumSize">
       <size>
        <width>511</width>
        <height>371</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>511</width>
        <height>371</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="Canvas" name="canvas" native="true">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>151</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>151</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="Canvas" name="canvas2" native="true">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>191</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>191</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="exitButton">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="aboutButton">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Social Media</string>
   </property>
  </action>
  <action name="actionTwitter">
   <property name="text">
    <string>Twitter</string>
   </property>
  </action>
  <action name="actionFacebook">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
  <action name="actionSnapchat">
   <property name="text">
    <string>Snapchat</string>
   </property>
  </action>
  <action name="actionInstagram">
   <property name="text">
    <string>Instagram</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTwitter_2">
   <property name="text">
    <string>Twitter</string>
   </property>
  </action>
  <action name="actionFacebook_2">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
  <action name="actionSnapchat_2">
   <property name="text">
    <string>Snapchat</string>
   </property>
  </action>
  <action name="actionLocal_Regional">
   <property name="text">
    <string>Local/Regional</string>
   </property>
  </action>
  <action name="actionNational">
   <property name="text">
    <string>National</string>
   </property>
  </action>
  <action name="actionInternational">
   <property name="text">
    <string>International</string>
   </property>
  </action>
  <action name="actionClearnet">
   <property name="text">
    <string>Clearnet</string>
   </property>
  </action>
  <action name="actionDarknet">
   <property name="text">
    <string>Darknet</string>
   </property>
  </action>
  <action name="actionDraw_network">
   <property name="text">
    <string>Draw network</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionData_Tables">
   <property name="text">
    <string>Data Tables</string>
   </property>
  </action>
  <action name="actionNetworks">
   <property name="text">
    <string>Networks</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header location="global">canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionTwitter</sender>
   <signal>changed()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1323</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
